src/components/tooltip/helpers/bv-tooltip.js

```diff
@@ -203,7 +203,7 @@ export const BVTooltip = /*#__PURE__*/ Vue.extend({
     this.$_hoverState = ''
     this.$_visibleInterval = null
     this.$_enabled = !this.disabled
-    this.$_noop = () => {}
+    this.$_noop = noop
```

Comment:
`this.$_noop = noop.bind(this)`

This handler needs a scope, so that only this instance of the event method is removed (not all that use the `noop` method)

Reply:
All fixed.